CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### [3.1.0+portage-3.1.4] - 2023-07-13
+
+### Changed
+
+ - Merged Release [V3.3.1](https://github.com/portagenetwork/roadmap/commit/47be9ba2102bd834bb55b0a95c2cec38a9cc0e35) into portage-deployment
```

Comment:
You can reference the upstream version that we merged: 3.3.1 https://github.com/DMPRoadmap/roadmap/releases/tag/v3.3.1

Reply:
Cool, I left all of that in the Unreleased portion and replaced the reference.